learn/cookbooks/running_production.mdx

```diff
@@ -3,106 +3,106 @@ title: Running Meilisearch in production — Meilisearch documentation
 description: Deploy Meilisearch in a Digital Ocean droplet. Covers installation,  server configuration, and securing your instance.
 ---
 
-# Running Meilisearch in production
+# Running a self-hosted Meilisearch project in production
 
-[Meilisearch Cloud](https://www.meilisearch.com/cloud?utm_campaign=oss&utm_source=docs&utm_medium=running-production-oss) is the recommended way to run Meilisearch in production environments. However, it is also possible to self-host Meilisearch.
+This tutorial will guide you through setting up a production-ready Debian server in a DigitalOcean droplet. It covers installing Meilisearch and setting up a secure environment.
```

Comment:
We could advertise about this marketplace Addon no?
<img width="825" alt="image" src="https://github.com/meilisearch/documentation/assets/4116980/8a744200-19fe-4936-8521-127984bff66c">

Reply:
This is a bit out of scope for the PR. My focus on this PR is to make sure all information is correct and to clean up this tutorial since I don't think anyone touched it in two years.

I've made a note of this for a future review, though!